### DISCUSSION TITLE
The new docking port sprite

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14565] [Unstable feedback]
Just no

![image](https://github.com/user-attachments/assets/992e4841-fd4b-4830-9504-2b6d454af0cf)

![image](https://github.com/user-attachments/assets/ab4aa5ff-0685-45ee-9301-276e0eaba3ae)

**Comment by JeroonBosch** [comment]
Yea that wasn't actually intended, this will be reverted

**Reply by ubertpendragon** [reply]
Can we get it as a wall sprite?
